src/Helpers.elm

```diff
@@ -0,0 +1,55 @@
+module Helpers exposing (..)
+
+{-| Returns the n first digits after the comma in a float
+
+    >>> nDigits 2 123.45
+    "45"
+
+    >>> nDigits 0 125
+    ""
+
+    >>> nDigits 1 1.99
+    "0"
+
+    >>> nDigits 2 1.0
+    "00"
+
+    >>> nDigits 2 -1.0001
+    "00"
+
+    >>> nDigits 2 0.01
+    "01"
+
+    >>> nDigits 2 0.10
+    "10"
+-}
+
+
+nDigits : Int -> Float -> String
+nDigits digits f =
+    let
+        multiplicator =
+            toFloat (10 ^ digits)
+
+        fint =
+            (round (f * multiplicator))
+    in
+        splitIntRec fint []
+            |> String.concat
+            |> String.right digits
+            |> String.padLeft digits '0'
+
+
+{-| Recursive helper to format an integer
+
+    >>> splitIntRec 12345 []
+    ["12", "345"]
+-}
+splitIntRec : Int -> List String -> List String
```

Comment:
This function has a terrible API. It always needs a empty list in the first call (why? the meaning of that is not clear , you can't guess by the function signature `splitIntRec : Int -> List String -> List String`; BTW the signature made me believe it was about the `List String`, not about the the `Int`)… Also I took a while to realize `splitIntRec` was `splitIntRecursively`, but that might be just me.

I strongly believe my old `addThousandSeparator` was way longer but it's signature and code was significantly more readable and meaningful.

If you really wanna stick to your solution, I'd explore the `List.foldr` or `List.foldl` functions instead.